src/components/ui/Toast/Toast.stories.tsx

```diff
@@ -0,0 +1,55 @@
+import { useUI, uiActions, uiEffects, uiInitialState } from 'src/sdk/ui'
+import { UIProvider } from '@faststore/sdk'
+
+import Toast from '.'
+
+export default {
+  component: Toast,
+  title: 'Atoms/Toast',
```

Comment:
Do you know why when testing this story on mobile, clicking on the button doesn't trigger the toast?
https://gatsby-store-storybook-git-feat-fsss-231-style-e45afa-faststore.vercel.app/?path=/story/atoms-toast--default&globals=backgrounds.grid:false
![CleanShot 2022-05-16 at 19 53 17](https://user-images.githubusercontent.com/381395/168695231-ca997f53-6b75-4a24-a1f2-c87a514ab6e1.gif)

Reply:
maybe it just failed punctually, I tested it here through this same link and it's ok

Reply:
It was my Chrome! My bad. :~

Reply:
I faced the same issue 👀 Not sure what happened. I'll try again.